sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -35,43 +35,32 @@ enum EntityType {
 }
 
 const alwaysBeExistingQueue = "alwaysbeexistingqueue";
-const alwaysBeDeletedQueue = "alwaysbedeletedqueue";
-
 const alwaysBeExistingTopic = "alwaysbeexistingtopic";
-const alwaysBeDeletedTopic = "alwaysbedeletedtopic";
-
 const alwaysBeExistingSubscription = "alwaysbeexistingsubscription";
-const alwaysBeDeletedSubscription = "alwaysbedeletedsubscription";
-
 const alwaysBeExistingRule = "alwaysbeexistingrule";
-const alwaysBeDeletedRule = "alwaysbedeletedrule";
 
 [
   {
     entityType: EntityType.QUEUE,
-    alwaysBeExistingEntity: alwaysBeExistingQueue,
-    alwaysBeDeletedEntity: alwaysBeDeletedQueue
+    alwaysBeExistingEntity: alwaysBeExistingQueue
   },
   {
     entityType: EntityType.TOPIC,
-    alwaysBeExistingEntity: alwaysBeExistingTopic,
-    alwaysBeDeletedEntity: alwaysBeDeletedTopic
+    alwaysBeExistingEntity: alwaysBeExistingTopic
   },
   {
     entityType: EntityType.SUBSCRIPTION,
     alwaysBeExistingEntity: alwaysBeExistingSubscription,
-    alwaysBeDeletedEntity: alwaysBeDeletedSubscription,
     parentTopicName: alwaysBeExistingTopic
   },
   {
     entityType: EntityType.RULE,
     alwaysBeExistingEntity: alwaysBeExistingRule,
-    alwaysBeDeletedEntity: alwaysBeDeletedRule,
     parentTopicName: alwaysBeExistingTopic,
     parentSubscriptionName: alwaysBeExistingSubscription
   }
 ].forEach((testCase) => {
-  describe(`Atom management - Basic CRUD on "${testCase.entityType}" entities #RunInBrowser`, function(): void {
+  describe.only(`Atom management - Basic CRUD on "${testCase.entityType}" entities #RunInBrowser`, function(): void {
```

Comment:
Remove .only please :)

This actually appears in multiple spots. Please remove all `only`s!

Reply:
Oops, change got excluded from local :/ - removed them